### PR TITLE
Remove sensitive error data

### PR DESCRIPTION
**Pull request by brunoabreu**
#### What is the purpose of this pull request?

Remove sensitive error data

#### What problem is this solving?

We are exposing sensitive data when an exception occurs: stack, headers, etc..

#### Screenshots or example usage
Now:
exposed

Fixed:
fixed_not_linked

Keeping information if the developer is linking the app:
fixed_linked

#### Types of changes

* [x] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: src/service/worker/runtime/graphql/utils/error.ts [hunk 1]
```diff
@@ -67,16 +68,18 @@ const formatError = (error: any, details?: any) => {
       formattedError.extensions.exception = {
         message: formattedError.originalError.message,
         name: formattedError.originalError.name,
-        stack: formattedError.originalError.stack,
-        ...formattedError.originalError,
+        ...omit(SENSITIVE_EXCEPTION_DATA, formattedError.originalError),
+        sensitive: pick(SENSITIVE_EXCEPTION_DATA, formattedError.originalError),
       }
     } else {
+      const mergedEx = {...formattedError.originalError,
```
Comment (tiagonapoli) [inline src/service/worker/runtime/graphql/utils/error.ts:75]: nit:
```suggestion
      const mergedEx = {
        ...formattedError.originalError,
```

Comment (tiagonapoli) [reply src/service/worker/runtime/graphql/utils/error.ts:75]: nit: How about we rename `mergedEx` -> `mergedException` -> I took some time to understand what that variable was

Comment (brunoabreu) [reply src/service/worker/runtime/graphql/utils/error.ts:75]: Right!

**Comment by brunoabreu** [comment]
Now it is working for HTTP requests (not GraphQL) too.

not_linked

**Pull request merged**